recipes/access.rb

```diff
@@ -60,3 +60,16 @@
   }]
   action :create
 end
+
+return unless node['platform_family'] == 'windows'
+
+if node['windows_hardening']['smbv1']['disable'] == true
```

Comment:
isn't [not_if](https://docs.chef.io/resource_common.html#not-if-examples) here a better option? I remember foodcritic complains about such ifs and suggests not_if/only_if